Alexa.NET.Tests/ResponseTests.cs

```diff
@@ -190,6 +192,544 @@ public void RepromptSsmlGeneratesPlainTextOutput()
             Assert.Equal(speech.ToXml(), ssmlText.Ssml);
         }
 
+        [Fact]
+        public void DeserializesExampleResponseJson()
+        {
+            var deserialized = Utility.ExampleFileContent<SkillResponse>("Response.json");
+
+            Assert.Equal("1.0", deserialized.Version);
+
+            var sessionAttributeSupportedHoriscopePeriods =
```

Comment:
At this stage we should be able to get supportedHoriscopePeriods as a JObject and then use .Value<bool> for the subsequent assertions.

The reason I say this isn't regarding the dynamic itself - more that using a dynamic allows a certain flexibility in the tests implementation; ideally that's something we want to avoid if people use the tests for information on expected behavior.

Reply:
updated to use JObject in cleanup commit